.github/workflows/test_pull_requests.yml

```diff
@@ -43,6 +43,25 @@ jobs:
           pip install check-manifest
       - name: Check Manifest
         run: check-manifest
+  localization_syntax:
+    # make sure all necessary files will be bundled in the release
+    name: Check l18n syntax
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+      - name: Check Manifest
+        run: |
+          pip install --upgrade pip semgrep
+          # f"..." and f'...' are the same for semgrep
+          semgrep --error --lang python --pattern 'trans._(f"...")' napari
+          semgrep --error --lang python --pattern 'trans._($X.format(...))' napari
```

Comment:
other patterns that could be used eventually are

trans._p(
trans._np(
trans._n(